src/main/tool_installers/updates/hudson.plugins.gradle.GradleInstaller

```diff
@@ -1,307 +1 @@
    "url": "https://services.gradle.org/distributions/gradle-1.0-rc-1-bin.zip"
  },
    {
    "id": "1.0-milestone-9",
    "name": "Gradle 1.0-milestone-9",
    "url": "https://services.gradle.org/distributions/gradle-1.0-milestone-9-bin.zip"
  },
    {
    "id": "1.0-milestone-8a",
    "name": "Gradle 1.0-milestone-8a",
    "url": "https://services.gradle.org/distributions/gradle-1.0-milestone-8a-bin.zip"
  },
    {
    "id": "1.0-milestone-8",
    "name": "Gradle 1.0-milestone-8",
    "url": "https://services.gradle.org/distributions/gradle-1.0-milestone-8-bin.zip"
  },
    {
    "id": "1.0-milestone-7",
    "name": "Gradle 1.0-milestone-7",
    "url": "https://services.gradle.org/distributions/gradle-1.0-milestone-7-bin.zip"
  },
    {
    "id": "1.0-milestone-6",
    "name": "Gradle 1.0-milestone-6",
    "url": "https://services.gradle.org/distributions/gradle-1.0-milestone-6-bin.zip"
  },
    {
    "id": "1.0-milestone-5",
    "name": "Gradle 1.0-milestone-5",
    "url": "https://services.gradle.org/distributions/gradle-1.0-milestone-5-bin.zip"
  },
    {
    "id": "1.0-milestone-4",
    "name": "Gradle 1.0-milestone-4",
    "url": "https://services.gradle.org/distributions/gradle-1.0-milestone-4-bin.zip"
  },
    {
    "id": "1.0-milestone-3",
    "name": "Gradle 1.0-milestone-3",
    "url": "https://services.gradle.org/distributions/gradle-1.0-milestone-3-bin.zip"
  },
    {
    "id": "1.0-milestone-2a",
    "name": "Gradle 1.0-milestone-2a",
    "url": "https://services.gradle.org/distributions/gradle-1.0-milestone-2a-bin.zip"
  },
    {
    "id": "1.0-milestone-2",
    "name": "Gradle 1.0-milestone-2",
    "url": "https://services.gradle.org/distributions/gradle-1.0-milestone-2-bin.zip"
  },
    {
    "id": "1.0-milestone-1",
    "name": "Gradle 1.0-milestone-1",
    "url": "https://services.gradle.org/distributions/gradle-1.0-milestone-1-bin.zip"
  },
    {
    "id": "0.9.2",
    "name": "Gradle 0.9.2",
    "url": "https://services.gradle.org/distributions/gradle-0.9.2-bin.zip"
  },
    {
    "id": "0.9.1",
    "name": "Gradle 0.9.1",
    "url": "https://services.gradle.org/distributions/gradle-0.9.1-bin.zip"
  },
    {
    "id": "0.9-rc-3",
    "name": "Gradle 0.9-rc-3",
    "url": "https://services.gradle.org/distributions/gradle-0.9-rc-3-bin.zip"
  },
    {
    "id": "0.9-rc-2",
    "name": "Gradle 0.9-rc-2",
    "url": "https://services.gradle.org/distributions/gradle-0.9-rc-2-bin.zip"
  },
    {
    "id": "0.9-rc-1",
    "name": "Gradle 0.9-rc-1",
    "url": "https://services.gradle.org/distributions/gradle-0.9-rc-1-bin.zip"
  },
    {
    "id": "0.9",
    "name": "Gradle 0.9",
    "url": "https://services.gradle.org/distributions/gradle-0.9-bin.zip"
  },
    {
    "id": "0.8",
    "name": "Gradle 0.8",
    "url": "https://services.gradle.org/distributions/gradle-0.8-bin.zip"
  },
    {
    "id": "0.7",
    "name": "Gradle 0.7",
    "url": "https://services.gradle.org/distributions/gradle-0.7-bin.zip"
  }
]}
```

Comment:
If you mean the order of the entries, this is the same behaviour the plugin/tool installation itself has. At some point/version the order changed.
Does it make sense to reorder the entries manually?